test/e2e/runner/rpc.go

```diff
@@ -92,18 +92,22 @@ func waitForNode(node *e2e.Node, height int64, timeout time.Duration) (*rpctypes
 
 // waitForAllNodes waits for all nodes to become available and catch up to the given block height.
 func waitForAllNodes(testnet *e2e.Testnet, height int64, timeout time.Duration) (int64, error) {
-	lastHeight := int64(0)
+	var lastHeight int64
+
 	for _, node := range testnet.Nodes {
 		if node.Mode == e2e.ModeSeed {
 			continue
 		}
-		status, err := waitForNode(node, height, 20*time.Second)
+
+		status, err := waitForNode(node, height, 30*time.Second)
```

Comment:
I was going to implement something that depended on the size of the network i.e. more nodes = longer wait

Reply:
Yeah, this should be configurable somewhat. Let's hold off on merging my PR until yours lands 👍